arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -1076,7 +1024,6 @@ ::frame_image( )
     }
 
     AVPixelFormat pix_fmt = static_cast<AVPixelFormat>(frame->format);
-    // If the pixel format is not recognized by then convert the data into RGB_24
```

Comment:
This comment is still useful, but is probably not clear or this might not be the best place for it.  This is really about the `else` case down on line 1075.  If we are unable to directly copy the pixels into one of our standard image memory layouts then we ask FFMpeg to convert it from whatever it was into RGB24.  So it's not that the pixel format is not recognized, it's that the pixel format is not a standard format for vital images.

Reply:
I added the back to the location you suggested.